CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.2)
+project(fzf)
+
+add_library(${PROJECT_NAME} SHARED "src/fzf.c")
+set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "lib")
+target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/src)
+
+if (CMAKE_C_COMPILER_ID STREQUAL "MSVC")
+	target_compile_options(${PROJECT_NAME} PRIVATE /W4)
+else ()
+	target_compile_options(${PROJECT_NAME} PRIVATE -Wall -std=gnu99)
```

Comment:
the project is actually c99 compatible, only tests need gnu99 (might be wrong in makefile as well)

We should rather add `set_property(TARGET fzf PROPERTY C_STANDARD 99)` because MSVC defaults to `11` i think.

Reply:
This would not matter for MSVC as it doesn't have C99 conformity. 
> The default C compiler (that is, the compiler when /std:c11 or /std:c17 isn't specified) implements ANSI C89, but includes several Microsoft extensions, some of which are part of ISO C99. Some Microsoft extensions to C89 can be disabled by using the [/Za](https://docs.microsoft.com/en-us/cpp/build/reference/za-ze-disable-language-extensions?view=msvc-170) compiler option, but others remain in effect. It isn't possible to specify strict C89 conformance. The compiler doesn't implement several required features of C99, so it isn't possible to specify C99 conformance, either.
So I am unsure as to what CMake's behaviour is in this case, however I can only assume it would then choose C11.
Though I agree this should be changed from gnu99 to c99.

Reply:
Uff. Sad life. I just was confused why CI said i should use c11 functions :laughing: